components/tyk-dashboard/templates/deployment-dashboard.yaml

```
@@ -0,0 +1,255 @@
+  name: dashboard-{{ include "tyk-dashboard.fullname" . }}
+  labels:
+    app: dashboard-{{ include "tyk-dashboard.fullname" . }}
+    chart: {{ include "tyk-dashboard.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  minReadySeconds: 5
+  strategy:
+    # indicate which strategy we want for rolling update
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 2
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      app: dashboard-{{ include "tyk-dashboard.fullname" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: dashboard-{{ include "tyk-dashboard.fullname" . }}
+        release: {{ .Release.Name }}
+      annotations:
+      # Enable backward compatibility to chart 0.8.1.
+      {{- if .Values.enableIstioIngress }}
+        traffic.sidecar.istio.io/excludeInboundPorts: "{{ .Values.containerPort }}"
+        traffic.sidecar.istio.io/includeInboundPorts: ""
+      {{- end }}
+        checksum/secrets: {{ include (print $.Template.BasePath "/secrets.yaml") . | sha256sum }}
+        {{- if .Values.mounts }}
+        {{- range $secret := .Values.mounts }}
+        "checksum/secret-{{ $secret.name }}": {{ include (print "mounts/%s" $secret.filename) . | sha256sum }}
+        {{- end }}
+        {{- end }}
+    spec:
+{{- if .Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.nodeSelector | indent 8 }}
+{{- end }}
+{{- if .Values.tolerations }}
+      tolerations:
+{{ toYaml .Values.tolerations | indent 8 }}
+{{- end }}
+{{- if .Values.affinity }}
+      affinity:
+{{ toYaml .Values.affinity | indent 8 }}
+{{- end }}
+      containers:
+      - image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        name: dashboard-{{ .Chart.Name }}
+        securityContext:
+          runAsUser: 999
+          runAsGroup: 999
+{{/*          fsGroup: 2000*/}}
+        env:
+          ## DEFAULT TYK DASHBOARD CONFIGURATION
+          - name: "TYK_DB_ENABLEOWNERSHIP"
+            value: "{{ .Values.config.enableOwnership }}"
+          - name: "TYK_DB_PAGESIZE"
+            value: "{{ .Values.config.defaultPageSize }}"
+          - name: "TYK_DB_REDISPORT"
+            value: "{{ .Values.redis.port }}"
+          - name: "TYK_DB_REDISHOST"
+            value: "{{ .Values.redis.host }}"
+          - name: "TYK_DB_NOTIFYONCHANGE"
+            value: "{{ .Values.config.notifyOnChange }}"
+          - name: "TYK_DB_HASHKEYS"
+            value: "{{ .Values.config.hashKeys }}"
+          - name: "TYK_DB_ENABLEDUPLICATESLUGS"
+            value: "{{ .Values.config.enableDuplicateSlugs }}"
+          - name: "TYK_DB_SHOWORGID"
+            value: "{{ .Values.config.showOrgId }}"
+          - name: "TYK_DB_ENABLEHOSTNAMES"
+            value: "{{ .Values.config.hostConfig.enableHostNames }}"
+          - name: "TYK_DB_DISABLEORIGSLUGPREFIX"
```

```suggestion
          - name: "TYK_DB_HOSTCONFIG_DISABLEORGSLUGPREFIX"
```
https://tyk.io/docs/tyk-dashboard/configuration/#host_configdisable_org_slug_prefix